src/Assignments/SimpleProblemPage.tsx

```diff
@@ -437,19 +437,22 @@ export const SimpleProblemPage: React.FC<SimpleProblemPageProps> = () => {
                     Object.keys(problemScores).map(key => {
                         return (
                             <div className="d-flex flex-row" key={key}>
-                                <div className="d-flex flex-column flex-grow-1">{(key === 'total') ? 'Total for this attempt' : `Problem #${key}`}</div>
+                                <div className="d-flex flex-column flex-grow-1">
+                                    {(key === 'total') && 'Total points (this attempt)'} 
```

Comment:
I feel like I would prefer this logic outside of the tsx
Do the if statements above and set a message variable and then just use that